src/plugins/Wallet/api.ts

```diff
@@ -626,7 +619,7 @@ export const gitcoinAPI = {
                     onTransactionError: reject,
                     onEstimateError: reject,
                 },
-            )
+            ).catch(reject)
```

Comment:
The error should have caught in the `sendTx` function. No need to catch it again.